taxonomies/categories.txt

```diff
@@ -60941,6 +60949,8 @@ fi:sardiineja oliiviöljyssä
 fr:Sardines à l'huile d'olive, sardines à l'huile d'olive vierge, sardines à l'huile d'olive vierge extra, sardines à l'huile d'olive égouttée
 nl:Sardientjes op olijfolie
 ro:Sardine în ulei de măsline
+agribalyse_proxy_food_code:en:26231
```

Comment:
There are exact matches for this category, it should be merged with the ciqual category that was added:

<en:Sardines in olive oil
<en:Sardine fillets
en:European pilchard or sardine in olive oil canned drained
fr:Sardine à l'huile d'olive appertisée égouttée
agribalyse_food_code:en:26040
ciqual_food_code:en:26040
ciqual_food_name:en:European pilchard or sardine, in olive oil, canned, drained

Reply:
(same for all the ciqual categories that just add "canned, drained")